ios/RCTPSPDFKit/RCTPSPDFKitViewManager.m

```diff
@@ -29,6 +34,11 @@ @implementation RCTPSPDFKitViewManager
     if (view.annotationAuthorName) {
       view.pdfController.document.defaultAnnotationUsername = view.annotationAuthorName;
     }
+
+    // Update the configuration to use the custom annotation tool bar.
+    [view.pdfController updateConfigurationWithBuilder:^(PSPDFConfigurationBuilder * _Nonnull builder) {
+      [builder overrideClass:PSPDFAnnotationToolbar.class withClass:CustomButtonAnnotationToolbar.class];
```

Comment:
The way this is currently implemented, this will use the custom annotation toolbar for **all** usages of the view component, not just for a specific example in the Catalog project. Was this intended?

Reply:
Yes, the intention is to use it all over the place.